static/templates/settings/account-settings.handlebars

```diff
@@ -103,7 +103,11 @@
                             <label for="old_password" class="title">{{t "Old password" }}</label>
                             <input type="password" autocomplete="off" name="old_password" id="old_password" class="w-200 inline-block" value="" />
                             <div class="info">
-                                <a href="/accounts/password/reset/" class="sea-green" target="_blank">{{t "Forgotten it?" }}</a>
+                                <a class="sea-green" id="forgot_password" target="_blank">{{t "Forgotten it?" }}</a>
+                                <a class="sea-green" reset_password_container id="reset_password" style="display: none">{{t "Reset password" }}</a>
+                                <div class="loading_indicator_spinner"></div>
+                                <a class="sea-green reset_password_container" id="resend_link">{{t "Resend" }}</a>
+                                <span id="email_text" style="display: none">{{t "(Check your email)" }}</span>
```

Comment:
I was being told that we don't use inline CSS in templates, so here, instead, you can use Bootstrap classes `.show` & `.hide` for controlling the display.